apps/morph_check

```diff
@@ -84,13 +84,15 @@ Default errors checked for
 * Zero radius points
 * Zero length segments
 * Zero length sections
+* Jumps in the Z-direction
 
 Default values for options
 --------------
 * has_nonzero_soma_radius 0.0
 * has_all_nonzero_neurite_radii: 0.007
 * has_all_nonzero_segment_lengths: 0.01
 * has_all_nonzero_section_lengths: 0.01
+* has_no_zjumps: 30 micrometer - distance considered a jump
```

Comment:
As we discussed, my plan is to:
1) Turn this off by default
2) allow one to pick the 'axis' - not everyone does reconstructions with slice in z axis, however, it should always (?) be one of x, y, z